src/tracer/ebpf/syscall_schema.h

```diff
@@ -82,7 +82,7 @@ namespace SyscallSchema
             // The line that contains this substring is up to where we skip to.
             std::regex syscall_nrRegex("__syscall_nr");
             // Regex to parse argument info.
-            std::regex argFieldTypeName("(field:)([a-zA-Z0-9_\\s\\*]+)+(;)");
+            std::regex argFieldTypeName("(field:)([a-zA-Z0-9_\\s\\*]+)(;)");
             // Regex to parse arg and name
             std::regex argTypeAndName("([a-z\\* _]+)+ ([a-z_]+)$");
```

Comment:
I believe this regex also suffers from the same issue.
```suggestion
            std::regex argTypeAndName("([a-z\\* _]+) ([a-z_]+)$");
```

On a separate note (no need to worry about this in this PR), I think this one and the other regex may be incomplete. They won't handle a line like

```
field:char comm[TASK_COMM_LEN]; offset:12;      size:16;
```
(taken from `/sys/kernel/debug/tracing/events/sched/sched_wakeup/format`)

due to the capital letters and the `[]` characters.

CC @jahabibi